app/src/main/java/com/x8bit/bitwarden/ui/auth/feature/masterpasswordguidance/MasterPasswordGuidanceScreen.kt

```diff
@@ -210,11 +214,13 @@ private fun TryGeneratorCard(
 @Composable
 private fun BulletTextRow(
     text: String,
+    modifier: Modifier = Modifier,
 ) {
     Row(
         Modifier
             .fillMaxWidth()
-            .padding(horizontal = 8.dp),
+            .padding(horizontal = 8.dp)
+            .then(modifier),
```

Comment:
I know we have to do this in some specific places but we usually just do this:

```kotlin
modifier = modifier
    .stuff()
    .and()
    .things(),
```

Thoughts on this?

Reply:
Something I was going to ask, then I searched for `.then()` and saw it a lot. But I don't have a firm stance, from the POV that order can matter IDK philosophically which is better I have usually done it the way you suggest here. But NAHTDO (not a hill to die on) esp since its just there for convention for rn.